src/Microsoft.Extensions.Options.ConfigurationExtensions/OptionsServiceCollectionExtensions.cs

```diff
@@ -23,6 +23,7 @@ public static IServiceCollection Configure<TOptions>(this IServiceCollection ser
             }
 
```

Comment:
The name of this class should mean "here are the extensions for adding option "configurations" that are based on values from `Microsoft.Extensions.Configuration`. If we go with `IOptionsConfiguration` for the interface, then the current name is a bit ambiguous. It should probably be revisited no matter what we do.